src/components/extendedTextField/ExtendedTextField.tsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-magic-numbers */
+import React, { useState } from 'react';
+import { Button, IconAngleDown, IconAngleUp } from 'hds-react';
+import { useTranslation } from 'react-i18next';
+
+type ExtendedTextFieldProps = {
+  content: string;
+};
+
+const ExtendedTextField = (props: ExtendedTextFieldProps) => {
+  const { content } = props;
+  const { t } = useTranslation();
+  const [expandable, setExpandable] = useState(false);
+  const [extended, setExtended] = useState(false);
+
+  if (content.length >= 350 && expandable === false) {
+    setExpandable(true);
+  }
+
+  const expandText = () =>
+    expandable && !extended ? setExtended(true) : setExtended(false);
+
+  return expandable ? (
+    <>
+      <p>{extended ? content : content.slice(0, 200)}</p>
```

Comment:
```suggestion
      <p>{extended ? content : content.slice(0, 350)}</p>
```
This should be longer and use the same value as above